yolact.py

```diff
@@ -609,6 +609,7 @@ def train(self, mode=True):
 
         if cfg.freeze_bn:
             self.freeze_bn()
+        return self
```

Comment:
Restoring the return arg of the base method. Otherwise, exporting with `ModuleExporter` fails.